root/components/relationship-editor.tt

```diff
@@ -47,11 +47,31 @@
     </div>
     <div>
       <label>
-        <input type="checkbox" data-bind="checked: $parent.changeAllRelationshipCredits" />
-        <span data-bind="html: MB.i18n.l('Change all relationship credits for {entity} on the page.', {entity: entity.html({target: '_blank'})})">
+        <input type="checkbox" data-bind="checked: $parent.changeOtherRelationshipCredits" />
+        <span data-bind="html: MB.i18n.l('Change credits to {entity} for other relationships on the page.', {entity: entity.html({target: '_blank'})})">
```

Comment:
The `entity` argument you're passing in doesn't have the new credit, so it looks like "Change credits to [the same name]."

I think there's a way to fix this which also makes the string easier to understand:

```JavaScript
MB.i18n.l('Change credits to “{credit}” for other {entity} relationships on the page.', {credit: relationship().creditField(source)(), entity: entity.html({target: '_blank'})})
```

Hopefully changing "other relationships" to "other {entity} relationships" makes it clear that even the options below, like "all relationships," are still restricted to relationships to that entity, not literally all relationships on the page.